csharp/sample/Microsoft.ML.OnnxRuntime.InferenceSample/Program.cs

```diff
@@ -6,7 +6,7 @@
 using System.Text;
 using System.IO;
 using Microsoft.ML.OnnxRuntime;
-using System.Numerics.Tensors;
+using Microsoft.ML.OnnxRuntime.Tensors;
```

Comment:
Do you think these couple new types need their own namespace? Would it be OK to just put the Tensor types in `Microsoft.ML.OnnxRuntime`?

Reply:
Kept it this way, I think it will be easier to switch, of add new api, if System.Numerics.Tensors is available.